gremlin-tools/gremlin-socket-server/conf/test-ws-gremlin.yaml

```diff
@@ -0,0 +1,35 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+PORT: 45943
```

Comment:
i think i see where you are going with this file, but could you clarify a bit why you factored out these identifiers from the java code to a yaml file?

Reply:
When writing tests for GLV's they will need to set these request ID's. I factored out into a YAML so they can be read from one place from any environment.

Reply:
yep, that's what i thought. i think you need documentation for how to use this thing. comments in this file would be a start, but a bit more javadoc in the `SimpleSocketServer` and related classes would help too. finally, i think you could use a more detailed treatment as a new section [here](https://tinkerpop.apache.org/docs/current/dev/developer/#developing-tests) and some notice of this new thing in the Upgrade Documentation in the provider section for drivers. Driver developers outside of TinkerPop could presumably use this thing, right?

Reply:
I'll add more documentation as you suggest. Any driver developer should be able to make use of this if they see fit.